flink-core/src/main/java/org/apache/flink/api/common/functions/util/AbstractRuntimeUDFContext.java

```diff
@@ -133,13 +132,24 @@ public DoubleCounter getDoubleCounter(String name) {
 		accumulators.put(name, accumulator);
 	}
 
-	@SuppressWarnings("unchecked")
```

Comment:
This should stay to prevent warnings, should also be added to the other `getAccumulator` method.